adobe_vendor_id.py

```diff
@@ -143,3 +178,59 @@ def handle_accountinfo_request(self, data, urn_to_label):
     def error_document(self, type, message):
         return self.ERROR_RESPONSE_TEMPLATE % dict(
             vendor_id=self.vendor_id, type=type, message=message)
+
+class AdobeVendorIDModel(object):
+
+    """Implement Adobe Vendor ID within the library registry's database
+    model.
+    """
+
+    def __init__(self, _db, node_value, temporary_token_duration=None):
+        self._db = _db
+        self.temporary_token_duration = (
+            temporary_token_duration or datetime.timedelta(minutes=10)
+        )
```

Comment:
You're right, the circ manager specifies the time the token _expires._ I'll fix the registry.